draft-ietf-quic-recovery.md

```diff
@@ -1142,6 +1142,21 @@ Pseudocode for OnPacketSent follows:
      SetLossDetectionTimer()
 ~~~
 
+## On Receiving a Datagram
+
+When a datagram is received, it may allow the server to send
+if the server was previously amplification limited, even if
+none of the packets in it can be processed.
```

Comment:
```suggestion
Receiving a datagram might allow the server to send additional data if the
server was previously blocked by the anti-amplification limits; see
{{before-address-validation}}. This is necessary even if none of the packets in
the datagram are successfully processed.
```

Reply:
I removed the reference because I couldn't find that section and removed "additional data", because the amplification factor applies to all packets, not just data or ack-eliciting packets.